test/e2e/errorcondition_test.go

```diff
@@ -115,6 +114,80 @@ func TestContainerErrorMsg(t *testing.T) {
 	// TODO(jessiezcc): add the check to validate that Route is not marked as ready once https://github.com/knative/serving/issues/990 is fixed
 }
 
+// TestContainerExitingMsg is to validate the error condition defined at
+// https://github.com/knative/serving/blob/master/docs/spec/errors.md
+// for the container crashing scenario.
+func TestContainerExitingMsg(t *testing.T) {
```

Comment:
IMO this should be under `./test/conformance`

Reply:
I agree, I'll move the whole file to the conformance suite, as the other tests are in the same league.